### PR TITLE
DOC: FutureWarning in Sphinx build when calling read_parquet

### LINKED ISSUE #32832: DOC: CI failure due to fsspec deprecation warning
There is CI failure in the ```Web/Docs```, under ```check ipython directive errors``` on github actions, noticed on #32831 and on #32830 

The error that is being shown:

```
Run ! grep -B1 "^<<<-------------------------------------------------------------------------$" sphinx.log
  FutureWarning,
<<<-------------------------------------------------------------------------
##[error]Process completed with exit code 1.
```

---

The root cause of the problem, is a warning that is generated during the creation of the docs.

```
>>>-------------------------------------------------------------------------
Warning in /home/runner/work/pandas/pandas/doc/source/user_guide/scale.rst at block ending on line 254
 Specify :okwarning: as an option in the ipython:: block to suppress this message
 ----------------------------------------------------------------------------
 /home/runner/miniconda3/envs/pandas-dev/lib/python3.7/site-packages/fsspec/implementations/local.py:33: FutureWarning: The default value of auto_mkdir=True has been deprecated and will be changed to auto_mkdir=False by default in a future release.
   FutureWarning,
<<<-------------------------------------------------------------------------
```

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [x] closes #32832
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Review by datapythonista — APPROVED** [review]
lgtm, thanks for the fix @MomIsBestFriend

**Comment by MomIsBestFriend** [comment]
@datapythonista Is there a reason to limit the line characters length to 80, in ```rst``` files?

**Comment by MomIsBestFriend** [comment]
Even if ```auto_mkdir=True``` is passed, it still produces the warning on the CI, but on my local machine passing ```auto_mkdir=True``` does solve this issue.

File: doc/source/user_guide/scale.rst [hunk 1]
```diff
@@ -249,7 +249,9 @@ We can use Dask's ``read_parquet`` function, but provide a globstring of files t
 
    import dask.dataframe as dd
 
-   ddf = dd.read_parquet("data/timeseries/ts*.parquet", engine="pyarrow")
+   ddf = dd.read_parquet(
+       "data/timeseries/ts*.parquet", engine="pyarrow", auto_mkdir=True
+   )
```
Comment (jorisvandenbossche) [inline doc/source/user_guide/scale.rst]: Another option is to set `:okwarning:` for now, as I suppose this is something to be fixed on the dask side

**Comment by jorisvandenbossche** [comment]
@MomIsBestFriend can you just merge master? I just merged another PR that fixes the theme name error you now get

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by MomIsBestFriend** [comment]
CI is green :)

**Comment by datapythonista** [comment]
Thanks for fixing this @MomIsBestFriend

**Pull request merged**